rmw_opensplice_cpp/src/rmw_get_encoding_identifier.cpp

```diff
@@ -0,0 +1,27 @@
+// Copyright 2014-2015 Open Source Robotics Foundation, Inc.
```

Comment:
Are the dates right? (might want to check the other related pr's I already approved too)